phpstan-baseline.neon

```diff
@@ -0,0 +1,6 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Class Sonata\\\\UserBundle\\\\Model\\\\User not found\\.$#"
```

Comment:
What is this new ?

Reply:
Oh, I see, you removed the `"sonata-project/user-bundle": "^3.6 || ^4.0"` dependency.
Should we keep it then ?

Reply:
We should keep it yes. The integration with the user-bundle is deprecated. I just removed the required-dev dependency in order to be able to test with Symfony 5.0